src/tools/sdk/archive/index.md

```diff
@@ -1,13 +1,14 @@
 ---
-title: "Index of Downloads"
-description: "Download specific stable and dev channel versions of the Dart SDK, Dartium, and the Dart API documentation."
-permalink: /install/archive
+title: Dart SDK Download
+short-title: Download
```

Comment:
Download -> Zip Files (in the title & short-title)
... or something else indicating that this isn't the primary download page.

Reply:
How about _Archive_?